packages/services/storage/src/index.ts

```diff
@@ -1849,9 +1837,8 @@ export async function createStorage(connection: string, maximumPoolSize: number)
           sv.has_persisted_schema_changes,
           sv.previous_schema_version_id,
           sv.composite_schema_sdl,
-          sv.schema_composition_errors,
-          sl.author,
-          lower(sl.service_name) as "service_name"
```

Comment:
@n1ru4l are you sure we don't need to expose `service_name`?

Reply:
yeah otherwise we would have a typescript/runtime error